networking/hardware_networks/configuring-sriov-device.adoc

```diff
@@ -16,8 +16,10 @@ include::modules/nw-sriov-configuring-device.adoc[leveloffset=+1]
 
 include::modules/nw-sriov-troubleshooting.adoc[leveloffset=+1]
 
-:FeatureName: CNI VRF plug-in
-include::modules/technology-preview.adoc[leveloffset=+1]
+[NOTE]
+====
+Using a VRF through the `ip vrf exec` command is not supported in OCP pods. Applications bind directly to the VRF interface to leverage VRF.
```

Comment:
Consider rephrasing the sentence to avoid using "leverage". Refer to the IBM Style Guide for guidance on "leverage" (Pg. 305 of the PDF version).
Suggestion: To use VRF, applications bind directly to the VRF interface.